### PR TITLE
SERPRO Assinador Serpro 4.0.2

### PR DESCRIPTION
**Pull request by dmirandaa**
SERPRO Assinador Serpro 4.0.2
 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/99508)

### CODE REVIEW

File: manifests/s/SERPRO/AssinadorSERPRO/4.0.2/SERPRO.AssinadorSERPRO.locale.pt-BR.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: SERPRO.AssinadorSERPRO
+PackageVersion: 4.0.2
+PackageLocale: pt-BR
+Publisher: SERPRO
+PackageName: Assinador SERPRO
```
Comment (Trenly) [inline manifests/s/SERPRO/AssinadorSERPRO/4.0.2/SERPRO.AssinadorSERPRO.locale.pt-BR.yaml:8]:
```suggestion
PackageName: Assinador Serpro
```
Should match control panel

File: manifests/s/SERPRO/AssinadorSERPRO/4.0.2/SERPRO.AssinadorSERPRO.locale.pt-BR.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+License: SERPRO
```
Comment (Trenly) [inline manifests/s/SERPRO/AssinadorSERPRO/4.0.2/SERPRO.AssinadorSERPRO.locale.pt-BR.yaml:9]:
```suggestion
License: Proprietário
```

**Review by Trenly — CHANGES_REQUESTED** [review]

File: manifests/s/SERPRO/AssinadorSERPRO/4.0.2/SERPRO.AssinadorSERPRO.locale.pt-BR.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: SERPRO.AssinadorSERPRO
+PackageVersion: 4.0.2
+PackageLocale: pt-BR
+Publisher: SERPRO- Serviço Federal de Processamento de dados
+PackageName: Assinador Serpro
+License: SERPRO
```
Comment (Trenly) [inline manifests/s/SERPRO/AssinadorSERPRO/4.0.2/SERPRO.AssinadorSERPRO.locale.pt-BR.yaml:9]: https://github.com/microsoft/winget-pkgs/pull/99508/files#r1140284091
The publisher is not a license. If it is not an open source license like GPL-2.0 or MIT, it should be proprietary

**Review by Trenly — CHANGES_REQUESTED** [review]

**Review by dmirandaa — COMMENTED** [review]
Changed the license and added a license URL

**Comment by stephengillie** [comment]
Hi @dmirandaa,

The application installed normally, but gave an error while launching in a sandbox:
image

[Error starting the new Subscriber services! If necessary, restart your computer.]

**Comment by stephengillie** [comment]
> Hi @dmirandaa,
> 
> The application installed normally, but gave an error while launching in a sandbox:
> 
> [Error starting the new Subscriber services! If necessary, restart your computer.]

Sorry for causing any concern - this turns out to be an issue with sandboxes being a reduced form of Windows. It appears that the services from the above screenshot are significant, as running in a full VM presents no issue: 
image

**Review by stephengillie — APPROVED** [review]

**Pull request merged**